test/cli/dune

```diff
@@ -0,0 +1,32 @@
+(executable
+ (name index_fsck)
+ (modules index_fsck)
+ (libraries index index.unix))
+
+(executable
+ (name generate)
+ (modules generate)
+ (libraries index index.unix))
+
+(rule
+ (alias generate-cli-test-data)
+ (targets data)
+ (package index)
```

Comment:
I'm not sure I understand why we need this option; why would this alias be build outside of the package?

Reply:
It's to distinguish it from the `index-bench` package, which doesn't need to generate this data as part of its build process.

Reply:
It's to distinguish it from the `index-bench` package, which doesn't need to generate this data as part of its build process.

Reply:
Thanks!